LICENSE

```diff
@@ -0,0 +1,143 @@
+Linux Agent for Operational Management Suite, ver 1.0 
+
+This Microsoft Open Source project incorporates material from Microsoft and the project(s) listed below (“Third Party Code”). Microsoft is not the original author of the Third Party Code. Microsoft reserves all other rights not expressly granted, whether by implication, estoppel or otherwise.   
```

Comment:
In general, I'm not wild on very long lines in README or LICENSE files. I prefer them to be 70 bytes or so per line (trivial to do in `emacs`). The issue is that, with very long lines, it's harder to identify a word change or spelling error in code reviews down the road.

Not worth changing for this; just keep in it in mind in the future.